src/WebJobs.Script/WebJobs.Script.csproj

```diff
@@ -39,8 +39,8 @@
   </ItemGroup>
   <ItemGroup>
     <!-- Dependencies needed for Storage Providers -->
-    <PackageReference Include="Azure.Core" Version="1.25.0" />
-    <PackageReference Include="Azure.Identity" Version="1.4.1" />
+    <PackageReference Include="Azure.Core" Version="1.34.0" />
+    <PackageReference Include="Azure.Identity" Version="1.9.0" />
```

Comment:
It may be advantageous to take the latest `1.10.0`

Reply:
Agreed. If we're going through an upgrade, let's make sure we snap to the latest minor of all of those dependencies.

Reply:
Changed to 1.10.0